tsconfig.js.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "./tsconfig.json",
+	"compilerOptions": {
+		"noImplicitAny": false,
```

Comment:
This is necessary as otherwise the js files would have loads of errors due to implicit anys